\hypertarget{class_security_1_1_entity_framework_1_1_exceptions_1_1_user_not_found_exception}{}\section{Класс Security.\+Entity\+Framework.\+Exceptions.\+User\+Not\+Found\+Exception}
\label{class_security_1_1_entity_framework_1_1_exceptions_1_1_user_not_found_exception}\index{Security.\+Entity\+Framework.\+Exceptions.\+User\+Not\+Found\+Exception@{Security.\+Entity\+Framework.\+Exceptions.\+User\+Not\+Found\+Exception}}


Исключение, возникает при неудачном поиске пользователя в коллекции базы данных  


Граф наследования\+:Security.\+Entity\+Framework.\+Exceptions.\+User\+Not\+Found\+Exception\+:\begin{figure}[H]
\begin{center}
\leavevmode
\includegraphics[height=2.000000cm]{d8/d01/class_security_1_1_entity_framework_1_1_exceptions_1_1_user_not_found_exception}
\end{center}
\end{figure}
\subsection*{Открытые члены}
\begin{DoxyCompactItemize}
\item 
\mbox{\Hypertarget{class_security_1_1_entity_framework_1_1_exceptions_1_1_user_not_found_exception_acbe74ac3bbade87bd08a8bc5fc4200df}\label{class_security_1_1_entity_framework_1_1_exceptions_1_1_user_not_found_exception_acbe74ac3bbade87bd08a8bc5fc4200df}} 
{\bfseries User\+Not\+Found\+Exception} (string login)
\end{DoxyCompactItemize}


\subsection{Подробное описание}
Исключение, возникает при неудачном поиске пользователя в коллекции базы данных 



Объявления и описания членов класса находятся в файле\+:\begin{DoxyCompactItemize}
\item 
C\+:/\+T\+F\+S2/\+Access\+Rights/\+Security.\+Entity\+Framework/\+Exceptions/User\+Not\+Found\+Exception.\+cs\end{DoxyCompactItemize}
